digital_land/commands.py

```diff
@@ -746,3 +746,48 @@ def process_data_in_batches(entities, flattened_dir, dataset_name):
         )
 
     return feature_collection
+
+
+def add_redirections(csv_file_path, pipeline_dir):
+    """
+    :param csv_file_path:
+    :param pipeline_dir:
+    :return:
+    """
+    expected_cols = [
+        "entity_HE",
+        "entity_LPA",
+    ]
```

Comment:
I presume these column names are for Historic England and the LPA? Could/should we use a more general name in case this process needs to be done again for something else? I'll take @eveleighoj here as he might have some input on that.

Reply:
Yes that makes sense!
Can we have something like "entity_source " and "entity_destination" ?